src/main/scala/io/confluent/examples/streams/WordCountScalaExample.scala

```diff
@@ -0,0 +1,134 @@
+package io.confluent.examples.streams
+
+import java.util.Properties
+import java.util.concurrent.TimeUnit
+
+import org.apache.kafka.streams.scala.StreamsBuilder
+import org.apache.kafka.streams.scala.kstream._
+import org.apache.kafka.streams.{KafkaStreams, StreamsConfig}
+
+/**
+  * Demonstrates, using the high-level KStream DSL, how to implement the WordCount program that
+  * computes a simple word occurrence histogram from an input text.
+  * Same as [[WordCountLambdaExample]] but in Scala.
+  *
+  * In this example, the input stream reads from a topic named "streams-plaintext-input", where the values of
+  * messages represent lines of text; and the histogram output is written to topic
+  * "streams-wordcount-output", where each record is an updated count of a single word, i.e.
+  * `word (String) -> currentCount (Long)`.
+  *
+  * Note: Before running this example you must 1) create the source topic (e.g. via `kafka-topics --create ...`),
+  * then 2) start this example and 3) write some data to the source topic (e.g. via `kafka-console-producer`).
+  * Otherwise you won't see any data arriving in the output topic.
+  *
+  *
+  * HOW TO RUN THIS EXAMPLE
+  *
+  * 1) Start Zookeeper and Kafka. Please refer to <a href='http://docs.confluent.io/current/quickstart.html#quickstart'>QuickStart</a>.
```

Comment:
This won't render nicely as JavaDocs -- not sure if we care?

Reply:
This part wasn't touched by the PR -- I'd rather have a separate PR if we want to update the style of the docs.

Reply:
Fair enough. But "not touched" ? This is a new example, so it was touched for sure :)